content/docs/04-clusters/04-palette-virtual-clusters/palette-virtual-cluster-quickstart.md

```diff
@@ -117,43 +117,41 @@ If **Load Balancer** is selected, the following must be true:<p></p><br />
 
 # Deploying a Palette Virtual Cluster
 
-1. From the slide menu, select the **Clusters** tab and click the **Palette Virtual Clusters** tab to list the available Palette Virtual Clusters, and then select **Add New Palette Virtual Cluster**.
+1. From the **Main** menu, click **Clusters** and select a project from the drop-down menu. 
 
+2. Click the **Virtual Clusters** tab to list the available Palette Virtual Clusters, and then select **Add New Virtual Cluster**.
 
-2. Complete the **Deploy New Palette Virtual Cluster** information:<p></p><br />
+3. Complete the **Deploy New Palette Virtual Cluster** information:<br />
 
-    - Select a Host Cluster.<p></p><br />
+    - Select a Host Cluster.<br />
 
     - Add a Cluster name.
 
-      **Note**: Use lowercase letters and do not add spaces.<p></p><br />
+      **Note**: Use lowercase letters and do not add spaces.<br />
```

Comment:
Notes should use the InfoBox component.  We also have large whitespaces between these elements that need to be reduced.

![CleanShot 2022-12-02 at 07 37 24](https://user-images.githubusercontent.com/29551334/205317300-814fbec3-c753-40b7-97d7-b3a2c52031b8.png)

Reply:
I wondered about that. I'll change it.